spellbook/models/tornado_cash/tornado_cash_deposits.sql

```diff
@@ -0,0 +1,237 @@
+{{ config(
+        alias ='deposits'
+        )
+}}
+
+SELECT *
+FROM
+(
+        -- Ethereum (ETH)
+        SELECT tc.evt_block_time AS block_time
+        , '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS currency_contract
```

Comment:
@hildobby we don't need to do this here since we are in a time crunch to merge this before the repo changes but I think this type of spell would be greatly improved by using dbt variables to set things like current contracts. 

eg at the top of the file something like 
{% set myVar = '2017-01-01' %}

that feels like it would make things more readable and easier for people to maintain and contribute to.